config.yaml

```
@@ -51,6 +51,9 @@ params:
       - label: Credits
         url: /credits
         style: link
+      - label: GSoC 2022
+        url: https://tardis-sn.github.io/gsoc/
+        style: link
```

```suggestion
        style: redirect-link
```
It's a separate website like tardis docs